### ISSUE TITLE
 Advanced Peripherals Compatability

### ISSUE THREAD

**Issue by Spuds1224** [neumond/python-computer-craft#12]
Is this project compatible with the [Advanced Peripherals](https://www.curseforge.com/minecraft/mc-mods/advanced-peripherals) mod?

**Comment by neumond** [comment]
No, I haven't made any wrappers for additional peripherals from other mods. Not sure if I can add & test them all, but I'll take a look.
